src/templates/Yarner-tex.toml

```diff
@@ -37,6 +37,11 @@ docs = "docs/"
 files = ["%%MAIN_FILE%%"]
 ## File(s) to include in code output (unprocessed), as glob pattern(s).
 # code_files = ["**/*.rs"]
+## Allows to modify paths of copied code files. Replaces path components. Optional.
+## When given, must have as many elements as code_files.
+## E.g., use "foo/bar" to replace the first two path component by foo and bar.
```

Comment:
Typo: "first two path components"

Concerning the design, I am somewhat skeptical about creating a dedicated mini language for this. If you go for the find-and-replace approach, regular expressions could do all of this and more while the reducing the cognitive overhead as most developers have used them in form or another.

Reply:
Personally, I am always struggling with RegEx and never really got it.

I would not see this as a mini-language and the only "special" here is the asterisk (which I may drop). Regarding the dot, the related part in the code is only for better readability of code output, the result would be the same (i.e. "." as "here"). Otherwise, it is just consecutive replacement of things between slashes.